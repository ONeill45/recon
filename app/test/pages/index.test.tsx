import { render } from '@testing-library/react'
import Index from '../../src/pages/index'

describe('Index page test', () => {
  it('should render index page', () => {
    const { queryByText } = render(<Index />)

    expect(queryByText('Welcome to Recon!')).toBeVisible()
  })
})
