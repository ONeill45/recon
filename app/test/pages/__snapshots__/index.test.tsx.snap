// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page test matches snapshot 1`] = `ShallowWrapper {}`;
